data/sgid-cadastre/plss/index.html

```diff
@@ -49,15 +49,15 @@
       <p>The State of Utah Public Land Survey System (PLSS) Fabric version 1.0 was generated from the Cadastral National Spatial Data Infrastructure (CadNSDI) version 2.0 which are the core or essential elements of cadastral information that provide the framework for building and using cadastral (land records) information nationwide.</p>
       <p>The PLSS data within the CadNSDI is typically survey data, control observations and other supplemental measurements, but in some cases survey information has not been completely abstracted or is not available and in these cases non Geographic Coordinate Database (GCDB)  information has been used such as digitized maps or data from other sources.  In all cases the data steward is identified in the data sets and the metadata records.</p>
-      <p>The PLSS Fabric is created and maintained through a collaborative partnership between the State of Utah Department of Technology Services, Automated Geographic Reference Center (AGRC); the State of Utah School and Institutional Trust Lands Administration (SITLA), and the United States Department of the Interior, Bureau of Land Management (BLM) and BLM contractor Premier Data Solutions (Premier) with contributions from the Utah Mapping and Information Partnership (UMIP) partnership through the Department of Environmental Quality (DEQ).</p>
-      <p>Survey Control added to the PLSS Fabric is provided by the BLM Surveyors, County Surveyors, County Contract Surveyors and Private Surveyors throughout the State. Survey Control can be added or viewed using the AGRC PLSS Web Application.<strong>
-        <em>Important Notes for PLSS Fabric Version 1.0:</em>
-      </strong>
-    </p>
+      <p>The PLSS Fabric is created and maintained through a collaborative partnership between the State of Utah Department of Technology Services, Automated Geographic Reference Center (AGRC); the State of Utah School and Institutional Trust Lands Administration (SITLA), and the United States Department of the Interior, Bureau of Land Management (BLM) and BLM contractor Premier Data Solutions (Premier) with contributions from the Utah Mapping and Information Partnership (UMIP) through the Department of Environmental Quality (DEQ).</p>
+      <p>Survey Control added to the PLSS Fabric is provided by the BLM Surveyors, County Surveyors, County Contract Surveyors and Private Surveyors throughout the State. Survey Control can be added or viewed using the <a href="http://mapserv.utah.gov/PLSS">>AGRC PLSS Web Application.</a></p>
+       <p><strong><em>Important Notes for PLSS Fabric Version 1.0:</em>
+      </strong></p>
     <ul>
-      <li>The Fabric works best with ArcMap 10.3.1 with Dec 15, <a href="http://support.esri.com/en/downloads/patches-servicepacks/view/productid/66/metaid/2252">2015 Patch</a>. Improvements will be in ArcMap 10.4.1.</li>
+      <li>The Fabric works best with ArcMap 10.4.1. Improvements will be in ArcMap 10.5 coming out in 2017.</li>
```

Comment:
I thought just the beta version is out or prerelease.....aka "will you test this for us"
leave as is for now.